src/actions/mod.rs

```diff
@@ -175,10 +176,12 @@ impl ActionHandler {
 
                         debug!("reload analysis: {:?}", project_path_clone);
                         let cwd = ::std::env::current_dir().unwrap();
-                        if let Some(new_analysis) = new_analysis {
-                            analysis.reload_from_analysis(new_analysis, &project_path_clone, &cwd, false).unwrap();
-                        } else {
+                        if new_analysis.is_empty() {
```

Comment:
After each crate compilation `rustc` can return an `Option` with analysis data and we collect unwrapped payloads into a vector, which we then load from file. For a single crate the behaviour is the same, since it's a change `None => vec![]` and `Some(analysis) => vec![analysis]`. However with this, now Cargo routine can collect and load the analysis data. There's one what-if: if a compiled indirect dependency will return `None` for analysis but there will be some collected into a vector, then a missing one won't be loaded from disk, since it'll only try to load the provided analysis in-memory, and possible cause subtle problems resolving the data? @nrc is such a case possible?